### DISCUSSION TITLE
Extending the default breakpoints

### DISCUSSION THREAD

**Discussion by mboisvertdupras** [barvian/fluid-tailwind#31] [Help] [answered]
This is more of a question than an issue but I was wondering how I could go about extending the screens provided by fluid-tailwind instead of overwriting them? For example, if I want to edit the 2xl breakpoint to be 100rem instead of 96rem.

**Comment by barvian** [comment, marked as answer]
Hi! You should be able to `extend` them like so:

```js
import fluid, { extract, screens } from 'fluid-tailwind'

export default {
  theme: {
    screens,
    extend: {
      screens: {
        '2xl': '100rem'
      }
    }
  }
  // ...
}
```

Let me know if that works as you'd expect.

**Reply by mboisvertdupras** [reply]
Works perfectly, thanks a bunch!
